testsuite/src/it/java/io/quarkus/ts/startstop/utils/Commands.java

```diff
@@ -675,6 +675,8 @@ public ProcessRunner(File directory, File log, List<String> command, long timeou
 
         @Override
         public void run() {
+            LOGGER.debugv("Running {0} in {1}", command, directory);
+            System.out.println("Running "+command+" in "+directory);
```

Comment:
Why are there no spaces between plus and vars/strings? Why do you prefer `System.outprintln` to logger info?